corehq/util/tests/test_global_request.py

```diff
@@ -21,3 +21,7 @@ def test_get_domain(self):
     def test_get_domain_null(self):
         set_request(None)
         self.assertEqual(None, get_request_domain())
+
+    def tearDown(self):
+        set_request(None)
+        super(GlobalRequestTest, self).tearDown()
```

Comment:
I think calling super  is only necessary for `tearDownClass` & `setUpClass`, but this is probably good proactice